client/src/components/Datepicker/styled/styled.ts

```diff
@@ -0,0 +1,77 @@
+import styled from 'styled-components';
+
+interface IStyledProps {
+    today?: boolean;
+    yesrerday?: boolean;
+    pickerPosition?: number | string;
+    hover?: boolean;
+    datePickerValue?: boolean;
+}
+
+export const DatePickerWrapper = styled.div`
+    width: 380px;
+`;
+
+export const DatePickerContainer = styled.div`
+    background-color: white;
+    box-shadow: 0px 5px 10px 2px rgba(34, 60, 80, 0.2);
+    width: 380px;
```

Comment:
how will it look on mobile devices?

Reply:
like this 
![image](https://user-images.githubusercontent.com/68894695/143874363-82d49369-3c00-4f02-9a8c-1f92ca2ab987.png)